pkgs/by-name/as/assetripper/package.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  stdenv,
+  autoPatchelfHook,
+  buildDotnetModule,
+  fetchFromGitHub,
+  dbus,
+  dotnetCorePackages,
+}:
+
+buildDotnetModule (finalAttrs: {
+  pname = "assetripper";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "AssetRipper";
+    repo = "AssetRipper";
+    tag = finalAttrs.version;
+    hash = "sha256-ixXWbygFhvOjld+YRLIhkO3cgDNkQsbivri2pjU4rgM=";
+  };
+
+  postPatch = ''
+    sed 's@Path.Join(ExecutingDirectory, "temp",@Path.Join(Path.GetTempPath(), "AssetRipper",@' \
+      -i Source/AssetRipper.IO.Files/Utils/TemporaryFileStorage.cs
+  '';
+
+  buildInputs = [
+    dbus
+    (lib.getLib stdenv.cc.cc)
+  ];
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  # Prevent automatic patching of all files. This is necessary as applying
+  # autoPatchelf indiscriminately causes dangling references to openssl and
+  # icu4c in AssetRipper.GUI.Free
+  dontAutoPatchelf = true;
+
+  # Avoid IOException on startup
+  makeWrapperArgs = [
+    "--add-flags"
+    "--log=false"
+  ];
+
+  # Make the main executable available under a more intuitive name.
+  postInstall = ''
+    mkdir -p $out/bin/AssetRipper
```

```suggestion
    mkdir -p $out/bin
```
Actually the `mkdir` is unnecessary because we assumed `AssetRipper.GUI.Free` exists.

---

The `mkdir` is necessary. As I said above, the symlink target hasn't yet been created by this phase.